.eslintrc.js

```diff
@@ -2,7 +2,6 @@
 module.exports = {
   extends: [
     'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
```

Comment:
@mluena why do you remove this line?

Reply:
because shadcn components complain, I would have to retype them all. And because I think it's misspelled (https://typescript-eslint.io/linting/configs/#projects-with-type-checking)

Reply:
What issues did you have?? I tried to add the plugin, and I don't see any typescript warn or error

Reply:
https://typescript-eslint.io/rules/no-unsafe-member-access/